Classes/Service/XsdGenerator.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+namespace SMS\FluidComponents\Service;
+
+use Sitegeist\FluidStyleguide\Domain\Model\Component;
```

Comment:
Werden die beiden Einträge überhaupt verwendet? Wir sollten wenn möglich vermeiden, in Components auf den Styleguide zu verweisen.

Reply:
Werden sie nicht, habe ich entfernt.